sdk/cli/bin/arv

```diff
@@ -85,7 +85,15 @@ def init_config
 end
 
 
-subcommands = %w(copy create edit keep pipeline run tag ws)
+subcommands = %w(copy create edit get keep pipeline run tag ws)
+
+def exec_bin bin, opts
+  @bin_path = `which #{bin}`.strip
```

Comment:
`@bin_path` should be just a local variable `bin_path` here. (aiui, `@bin_path` is an instance variable of "main", aka a global, and although practically speaking it looks harmless, globals do have a way of becoming harmful over time, so probably best to fix it now.)